spoon-pom/pom.xml

```diff
@@ -52,8 +52,14 @@
         <version>5.7.2</version>
         <scope>test</scope>
     </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <version>3.10.0</version>
+            <scope>test</scope>
+        </dependency>
 
-  <dependency>
+        <dependency>
```

Comment:
Unnecessary whitespace change, not related to your patch. The indentation in this file is pretty off but that's for a separate PR.